apps/coordinator/ts/health/health.service.ts

```diff
@@ -121,25 +121,22 @@ export class HealthService {
    * @returns the address and its funds in all networks
    */
   async checkWalletFunds(): Promise<ICheckWalletFunds> {
-    let address: string;
-    const networks = Object.values(ESupportedNetworks);
-
-    try {
-      const signer = getSigner(networks[0]);
-      address = await signer.getAddress();
-    } catch (error) {
+    if (!process.env.PRIVATE_KEY && !process.env.MNEMONIC) {
```

Comment:
I like this approach (we are covering the PRIVATE KEY in the .env file but `const signer = getSigner(networks[0])` does not throw an error when a network is not configured? I don't remember but I think there could be more reasons for `getSigner()` to throw than not having a PRIVATE KEY in the .env

Reply:
I removed top level call of `getSigner` and it will be handled inside `fundsInNetworks`